DocumentedExamples/Zonally_Averaged_Global_Meridional_Overturning_Circulation.ipynb

```diff
@@ -6,9 +6,9 @@
    "source": [
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #12.    plt.title('Overturning in %s' % getvar_args['expt']);
</pre><p><code>plt.title("Overturning in {}".format(getvar_args["expt"])</code> I think it's a bit nicer to use newer-style string formatting. But <code>getvar_args</code> might change here anyway.  </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/COSIMA/cosima-recipes/pull/217/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>